NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -151,6 +152,18 @@ class NeuroID
                 this.getApplicationContext()
                     ?.let { nidCallActivityListener?.setCallActivityListener(it) }
             }
+
+            // get current connected status
+            application?.let {
+                val connectivityManager =
+                    it.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+                val info = connectivityManager.activeNetworkInfo
+                val isConnectingOrConnected = info?.isConnectedOrConnecting()
+                if (isConnectingOrConnected != null) {
+                    isConnected = isConnectingOrConnected
+                }
+                isWifi = info?.type == ConnectivityManager.TYPE_WIFI
+            }
```

Comment:
This isn't related to the FPJS work is it?

Reply:
Nope but this was not there originally and the latency need this info otherwise the wifi state is not accurate. I can move this out to a seperate PR, get the network info initialization merged first, before merging this PR.